docs/compute-engine/build/arm.md

```diff
@@ -0,0 +1,153 @@
+    `//larq_compute_engine/tflite/benchmark:lce_benchmark_model`
+
+    The resulting binaries will be stored at
+    `bazel-bin/examples/lce_minimal` and
+    `bazel-bin/larq_compute_engine/tflite/benchmark/lce_benchmark_model`. You can
+    copy these to your ARM machine and run them there.
+
+=== "Building with Make"
+
+    To build LCE with Make, first clone the Larq Compute Engine repository and make
+    sure the tensorflow submodule is loaded:
+
+    ```bash
+    git submodule update --init
+    ```
+
+    To simplify the build process for various supported targets, we provide the
+    `build_lce.sh` script which accepts the build target platform as an input
+    argument.
+
+    To natively build the LCE library and C++ example programs, first you need to
+    install the compiler toolchain on your target device.
+
+    === "Linux (Debian based)"
+        On Debian based systems like a
+        Raspberry Pi board with Raspbian, run the following command:
+
+        ```
+        sudo apt-get install build-essential
+        ```
+
+    === "Linux (Arch based)"
+        On an Arch based system like a Raspberry Pi board with Manjaro operating system, run the following command instead:
+
+        ```
+        sudo pacman -S base-devel
+        ```
+
+    You should then be able to natively compile LCE by running the following from
+    the LCE root directory:
+
+    ```bash
+    larq_compute_engine/tflite/build_make/build_lce.sh --native
+    ```
+
+    It is also possible to replace `--native` by `--rpi` (32-bit ARM) or
+    `--aarch64` (64-bit ARM) to add extra compiler optimization flags.
+
+    The resulting compiled files will be stored in `gen/<TARGET>/` where,
+    depending on your target platform, `<TARGET>` can be `linux_x86_64`,
+    `rpi_armv7l`, or `linux_aarch64`. Here you can find the example program
+    `lce_minimal` and benchmark program `lce_benchmark`.
+
+
+    !!! note
+        On some systems the compiler is incorrectly named `aarch64-unknown-linux-gnu-gcc`
+        while it should be named `aarch64-linux-gnu-gcc`. If building with the option
+        `--aarch64` results in errors then the following bash script can be used to
+        create symlinks that fix this naming issue.
+
+        ```bash
+        #!/usr/bin/env bash
+
+        cd /usr/bin
+        for unknownfile in aarch64-unknown-linux-gnu-*; do
+        	newfile="${unknownfile/-unknown-/-}"    
+        	echo "Creating symlink $newfile that points to $unknownfile"
+        	ln -s $unknownfile $newfile
+        done
+        ```
+
+=== "Cross-compiling with Make"
+
+    First clone the Larq Compute Engine repository and make sure the tensorflow
+    submodule is loaded:
+
+    ```bash
+    git submodule update --init
+    ```
+
+    To cross-compile LCE, you need to first install the compiler toolchain:
+
+    === "Linux (Debian based)"
+        ```bash
+        sudo apt-get update
+        sudo apt-get install crossbuild-essential-armhf
```

Comment:
Don't we need `crossbuild-essential-arm64` for 64-bit ARM based systems?

Reply:
Nice catch, you're likely right. I can't test this stuff myself, only the Arch variants.